documentation/docs/01-routing.md

```diff
@@ -48,27 +48,27 @@ A file or directory can have multiple dynamic parts, like `[id]-[category].svelt
 Endpoints are modules written in `.js` (or `.ts`) files that export functions corresponding to HTTP methods. For example, our hypothetical blog page, `/blog/cool-article`, might request data from `/blog/cool-article.json`, which could be represented by a `src/routes/blog/[slug].json.js` endpoint:
 
 ```ts
-type Request<Context = any> = {
-	host: string;
+type Request<Locals = Record<string, any>> = {
 	method: 'GET';
```

Comment:
I think this should be `string`

Reply:
Since this is initially specified as `'GET'`, should we just limit it to only methods in https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods? Will there be a case where `method` isn't any of these or possibly a custom one?

Though, this is probably better implemented in another PR together with the types changes.

Reply:
that sounds somewhat reasonable, but yeah we'd have to change the implementation as well and agree that'd be better done as a separate PR